### DISCUSSION TITLE
2.3.61 - Error on Kibana Administration page

### DISCUSSION THREAD

**Discussion by abesinger** [Security-Onion-Solutions/securityonion#4942] [Unsupported Versions] [answered]
Hi,

I am running 2.3.61 (upgraded this weekend), enabled elastic-auth, can login to Elastic using my credential, but when I go to the Administration page, I get this error:

> You do not have the required Kibana permissions to use Elastic Security Administration.
> It looks like Fleet is disabled. Fleet must be enabled to use this feature. If you do not have permissions to enable Fleet, contact your Kibana administrator.

My account has the superuser role (I verified with an Elastic query); I should have permission to access everything, correct?

Addiitonally, I don't understand the "Fleet is disabled" error.  As far as I can tell, Fleet is running, I can access the FleetDM page. And if it *is* disabled, how do I enable it?

TIA,

-- Abe

**Comment by TOoSmOotH** [comment]
Elastic Security Agent is not supported. The fleet that it is mentioning is the fleet inside of Kibana that is used to manage all of the agents.

**Comment by abesinger** [comment]
So how are we supposed to view/modify roles for Elastic users? And FWIW, I was able to access the Kibana Administration page *before* enabling Elastic Auth.

**Comment by abesinger** [comment, marked as answer]
I retract the discussion, it was my mistake.  I should have been trying Stack Management.  I'll blame it on being late at night. :-)
